src/web.rs

```diff
@@ -45,6 +46,10 @@ impl Handler for SysinfoIronHandler {
                     Ok(Response::with((status::Ok,
                                        "text/html".parse::<Mime>().unwrap(),
                                        INDEX_HTML)))
+                } else if *path == "favicon.ico" {
+                    Ok(Response::with((status::Ok,
+                                       "text/html".parse::<Mime>().unwrap(),
```

Comment:
I think we can use correct mime type here, it's: `image/x-icon` for our favicon.ico file.

@GuillaumeGomez can you see this?